hpx/runtime/runtime_fwd.hpp

```diff
@@ -15,15 +15,16 @@
 namespace hpx
 {
     class HPX_API_EXPORT runtime;
-
-    ///////////////////////////////////////////////////////////////////////////
-    class HPX_API_EXPORT runtime_impl;
+    class HPX_API_EXPORT runtime_distributed;
```

Comment:
This probably needs to be declared only if `HPX_HAVE_DISTRIBUTED_RUNTIME` is set.